client/landing/gutenboarding/onboarding-block/create-site/index.tsx

```diff
@@ -35,7 +35,7 @@ const CreateSite: FunctionComponent< {} > = () => {
 				</div>
 				<div className="create-site__content">
 					<div className="create-site__placeholder create-site__placeholder-title">Placeholder</div>
-					<div className="create-site__text">{ NO__( 'Your site is being created...' ) }</div>
```

Comment:
Three dots to ellipsis `...` -> `…`

Reply:
Oh, it's there. Just not covered because of the `NO` prefix 🙂 

https://github.com/Automattic/wp-calypso/blob/bfd109a184277948e8218272d12885fc3a6ad71c/packages/eslint-plugin-wpcalypso/lib/rules/i18n-ellipsis.js#L61-L75